modules/apps/transfer/keeper/migrations.go

```diff
@@ -34,6 +36,14 @@ func (m Migrator) MigrateTraces(ctx sdk.Context) error {
 				iterErr = err
 				return true
 			}
+
+			if dt.IBCDenom() != newTrace.IBCDenom() {
+				// The new form of parsing will result in a token denomination change
+				// a bank migration is required. A panic should occur to prevent the
```

Comment:
```suggestion
				// The new form of parsing will result in a token denomination change.
				// A bank migration is required. A panic should occur to prevent the
```